.circleci/config.yml

```diff
@@ -245,6 +245,10 @@ aliases:
     name: iOS End-to-End Test Suite
     command: node ./scripts/run-ci-e2e-tests.js --ios --retries 3;
 
+  - &run-js-e2e-tests
+    name: JavaScript End-to-End Test Suite
+    command: node ./scripts/run-ci-e2e-tests.js --js --retries 3;
+
   - &display-broken-tests-warning
```

Comment:
This is unrelated to your PR but if you can tackle it while you're touching this: can you remove this alias? It's no longer used.